modules/promotion/commerce_promotion.post_update.php

```diff
@@ -84,3 +84,17 @@ function commerce_promotion_post_update_4() {
     $promotion->save();
   }
 }
+
+/**
+ * Remove promotion current_usage field.
```

Comment:
"Remove the promotion current_usage field."
People will see the lack of an article and think a Slavic person wrote this :P